service/src/main/java/bio/terra/policy/db/PaoDao.java

```diff
@@ -396,36 +391,32 @@ private Map<String, PolicyInputs> getAttributeSets(List<String> setIdList) {
         SELECT set_id, namespace, name, properties, conflicts
         FROM attribute_set
         WHERE set_id IN (:set_id_list)
```

Comment:
nit: set_ids?

Reply:
I don't think so. Each row in the table is from a single set - not plural.